src/components/MapsSection/MapsSection.styles.tsx

```diff
@@ -33,7 +33,7 @@ export const BoxWrapper = styled.div`
   justify-content: center;
   gap: 1rem;
   max-width: 1440px;
-  height: 550px;
+  height: 650px;
```

Comment:
I believe that the height of 550px is better for reasons of standardization with the other sections, in addition to keeping the layout smaller, more reduced. By covering the entire page, we have a very strong focus on previewing
```suggestion
  height: 550px;
```

Reply:
sure! i'll fix that. thanks.